share/pot/clone.sh

```diff
@@ -19,6 +19,8 @@ clone-help()
 	echo '  -i ipaddr : an ip address or the keyword auto (if applicable)'
 	echo '  -B bridge-name : the name of the bridge to be used (private-bridge only)'
 	echo '  -S network-stack : the network stack (ipv4, ipv6 or dual)'
+	echo '  -d dns : change dns to one of: inherit, pot, off or custom:filename'
+	echo '           the latter case will copy the file in the pot configuration'
```

Comment:
```suggestion
	echo '  -d dns : change dns resolver to one of:'
	echo '           inherit       - inherit from jailhost (default)'
	echo '           pot           - the pot configured in POT_DNS_NAME'
	echo '           custom:<file> - copy "file" into pot configuration'
	echo '           off           - leave resolver config unaltered'
```

Reply:
I had no idea what `-d pot` does (really nice feature actually).

Reply:
yes, the idea was to have one or few "system" pots that could enrich the eco-system.
The first was DNS, with donsmasq + consul, could automagically manage the DNS registry of pot's and services.
This feature became more and more complex to maintain, so I'm not suggesting it.